src/tester.tsx

```diff
@@ -111,30 +118,34 @@ class Tester {
     return this.component.text();
   }
 
-  find (selector) {
+  find (selector: string) {
     return this.wrapper.find(selector);
   }
 
   update () {
     return this.wrapper.update();
   }
 
-  async sleep (ms) {
+  async sleep (ms?: number) {
     await sleep(ms);
   }
 
-  async refresh (ms) {
+  async refresh (ms?: number) {
     await sleep(ms);
     this.update();
   }
 
   createShallowWrapper () {
     this.shallow = {};
-    this.shallow.wrapper = this.config.enzyme.mount(<this.TestedComponent.wrappedComponent {...this.props} { ...this.AppState } />);
+    const WrappedComponent = (this.TestedComponent as any).wrappedComponent as ComponentClass;
```

Comment:
👏  I honestly think that I would have spent 2 hrs trying to figure that one out and ended up asking you for help. gj !